src/modules/getthis/components/GetThisOptionList/index.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import config from '../../../../config'
+import { DetailsList } from '../../../core'
+import GetThisOption from '../GetThisOption'
+
+class GetThisOptions extends React.Component {
+  render() {
+    const { record } = this.props
+
+    if (record && record.getthis) {
+      const { status, options } = record.getthis
+
+      if (status === 'Success') {
+
+        if (options.length === 0) {
+          return (
+            <section className="card get-this-section">
+              <h2 className="get-this-section-heading">How would you like to get this item?</h2>
+
+                <div className="alert alert-warning">
+                  <p><b>Error:</b> No options available.</p>
+                </div>
+            </section>
+          )
+        }
+
+        return (
+          <section className="card get-this-section">
+            <h2 className="get-this-section-heading">How would you like to get this item?</h2>
+
+            <DetailsList className="get-this-options">
+              {options.map((option, key) => (
+                <GetThisOption option={option} key={key} />
+              ))}
+            </DetailsList>
+          </section>
+        )
+      } else if (status === 'Not logged in') {
+        let loginRoot = config.loginUrl.development
+
+        if (process.env.NODE_ENV === 'production') {
+          loginRoot = config.loginUrl.production;
+        }
```

Comment:
Is there a way for loginRoot to always have the right value?  Maybe like:
```
const loginRoot = config.loginUrl[process.env.NODE_ENV] || config.loginUrl.development;
```
Actually, could that be used while setting loginUrl?   I'll add that up there.

Reply:
I'll use this suggestion. It's cleaner.